poolup-devs/poolup_backend
# Update Login/Signup

Changelog 
- Set the 30 min limit for each verification link sent
- The user can return to fill out their account info using an unexpired verification link
- Bio is mandatory and we continuously direct them to that page until they fill it out 

User model 
- Remove all instances of name -> firstName and lastName 
- Remove all instances of verified -> isRegistered
   - all users in the database are verified; they are registered when they create a password
- Create /users/sendVerificationEmail 
- Update /users/verify endpoint to handle token expiration, preexisting registered accounts, and resuming sign-up session 
- Update /users/signup endpoint 
- Update /users/login endpoint 
- Create /users/get-about-me 

Review model 
- Refactor addNewReview to use makeReviewPublic 
- Use firstName for notifications asking users to leave reviews 

Bugfixes
- Fix race condition with cancellation of rides 
- Fix the case where one unit test can cause several others to fail by avoiding the use of findById
- Fix bug with scheduler testing
